prow/sealedsecret-docker-config.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: bitnami.com/v1alpha1
+kind: SealedSecret
+metadata:
+  creationTimestamp: null
+  name: docker-config
+  namespace: default
+spec:
+  encryptedData:
+    config.json: AgAvo/F1wEGUktcY44GKHwkkE34XGck/KAclDU0kcrMv0WHAKL73ZAKoC5hLhqCIblZ4GLsINizPDsZ0+4ui5kwuH72vobYWS1wGpkND1wKh8q0wB4XmEv/5yLB/1vWp827C9dhqOcZU1ekF+nvEmK5IElgwnaSeEJy3wEhjHz6Xr2uvyp32TtSld3OgpZXEmhNWSptSIvSX0GGPVGGkmVirp+6zJLSz1TGG6QZC/sr3MtBVqeNJ+eHuEo2v8/KgL9Qk/99VxZwVVvlk0r2RyDv6mrScMeyH893BZaArnirXyxWez8XSfTs2cCVFnCBy0KBeSNlMSWydbi2jVLDUtSpqqNIE5wrle+WAqglHrRcOt4cT2uoGBX8GMuf9Tna+b4D4B37Dt6k+shJqJzTaD4qACAgvl9XhcM4kjLS+qTsutLxTWIre0nB1RydZ0xH56pshA054Oyp2UB8io6yUnHToZE04jsM0j6BLDiIMAHYSwmV5Q+En/4kL4TqRahBL0HLRtPPwQB6q7yA32W2iZuOqMcFCQg29t0bdb/om+5BKP3BKxf7IBoWTYLhD7IsWLG1ZrOBN8P6su0tA6+0Qut9z2y66lhncOr4hHRTunk9JHq/6vGtwlBL0L9XVAHaxTqXA6LRiGecgRJ/XVGLG790zDjADfs8q6cSLiRP7oAWH1gn+3ylZwx+djbqCAQSTu0UOv1mI4cON6NsFxwx5avAO7lco8+ah/wvqR3i07sQSekYmOglsJcdCGob+YvEIErfawmDATuuG1rEf0frU+fAtTdBt71zDjJLCDo/msgY2Wi5+3Tr9kdzUGA3s3WA9kRVGdQUNzrKh4F+fKPBN6KNqZ1qn9B8uc9omnePZnsYCSvaMvpZgF3c6WnryQBRMeYkO1ceNvDRLpoI4hi0Ts941GJLqaMA0KaIEBHaPAF3Bg1sBHHgQRCR81QmTGmv4YyHomi0IV500yccoOXMfD9Boxtjqlr5MqRyZ3k5tBjBW13dmHrBPvWd6LTE3Gxkkj59kiK7uqqfD2JqvqyK7gwBH3mZsFIwFEKQD9I7pztvWVjvXu6oTkgih3nKkOLiT52COabWRnKHKxqZ3E3kGsQ8pKZul8CYYx8jexHokOWGHVNa22rkG81N8
+  template:
+    metadata:
+      creationTimestamp: null
+      name: docker-config
+      namespace: default
+    type: Opaque
+status: {}
+
```

Comment:
Nitpick; I'd get rid of the `creationTimestamp: null` and `status: {}` as they're not necessary.

Reply:
Done in https://github.com/pusher/testing/pull/107/commits/518a47fcc9a4b0203577667ca636f821c57127fa